### DISCUSSION TITLE
What build step builds the gnome-shell-theme.gresource files?

### DISCUSSION THREAD

**Discussion by schuelermine** [ubuntu/yaru#3760] [Q&A] [answered]
I’m trying to manually build Yaru but my builds end up missing a few files that are present in the Ubuntu package.
What is this project doing differently to generate the files share/gnome-shell/gnome-shell-theme.gresource and share/gnome-shell/theme/Yaru/gnome-shell-theme.gresource?

My current build process is:
$ meson ./build --prefix $(realpath ./out)
$ cd build; ninja install

**Comment by Muqtxdir** [comment, marked as answer]
you'd need to enable the gresource option at build-time for that
